lib/Conversion/TorchToMhlo/TorchToMhlo.cpp

```diff
@@ -49,7 +51,8 @@ class ConvertTorchToMhlo : public ConvertTorchToMhloBase<ConvertTorchToMhlo> {
     MLIRContext *context = &getContext();
     ConversionTarget target(*context);
     target.addLegalDialect<chlo::ChloDialect, mhlo::MhloDialect,
-                           tensor::TensorDialect, arith::ArithDialect>();
+                           stablehlo::StablehloDialect, tensor::TensorDialect,
```

Comment:
Nit: The naming MHLO now could be renamed to StableHLO.

Reply:
Fixed.